src/WarcraftLegacies.Source/GameLogic/GameEnd/ControlPointVictory.cs

```diff
@@ -59,7 +59,8 @@ private static void TeamWarning(Team whichTeam, int controlPoints)
 
     public static void Setup()
     {
-      //ControlPoint.OnControlPointOwnerChange += ControlPointOwnerChanges;
+      foreach (var controlPoint in ControlPointManager.Instance.GetAllControlPoints())
+        controlPoint.ChangedOwner += ControlPointOwnerChanges;
```

Comment:
That's really funny. 
Could you also reduce the CP count requirements? Not sure what would work better, just have a guess for now.